### ISSUE TITLE
Add possibility to change configuration of the game before the next round

### ISSUE THREAD

**Issue by Damgermys** [MorelGames/pitit-bac#28]
Can be nice to be able to:
- add or remove a question 
- adapt the time by round
- add/remove remain rounds
- ...
between 2 round without losing the points of the first rounds.

**Comment by AmauryCarrade** [comment]
Agreed. That could be not too hard to implement, actually. I'll look into that :)
